providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/GrpcConnector.java

```diff
@@ -129,9 +129,9 @@ private void observeEventStream() {
 
             ServiceGrpc.ServiceStub localServiceStub = this.serviceStub;
 
-            if (this.streamDeadlineMs > 0) {
-                localServiceStub = localServiceStub.withDeadlineAfter(this.streamDeadlineMs, TimeUnit.MILLISECONDS);
-            }
+            // if (this.streamDeadlineMs > 0) {
+            //     localServiceStub = localServiceStub.withDeadlineAfter(this.streamDeadlineMs, TimeUnit.MILLISECONDS);
+            // }
```

Comment:
This for some reasons, breaks the reconnect logic. I have no idea why. I'm not sure if it's a bug in gRPC, or elsewhere. I can't make any sense of it.